dockertest/output_unittests.py

```diff
@@ -156,7 +156,7 @@ def test_client_new(self):
         self.assertEqual(docker_version.client, '1.8.2')
         self.assertEqual(docker_version.server, '1.8.2')
 
-    def test_client_new(self):
+    def test_client_info_new(self):
```

Comment:
Expand the context, so you can see that this used to have the same name as the previous function. Hence one of them was masked.